bin/layers.go

```diff
@@ -47,10 +37,8 @@ var LayerUsage cli.Command = cli.Command{
 	Arguments: []cli.Argument{&cli.StringArg{Name: "url", Min: 1, Max: 1, Values: &url, UsageText: ""}},
 	Action: func(ctx context.Context, c *cli.Command) error {
 		sysCtx := types.SystemContext{}
-		out, err := ShowLayerUsage(url[0], ctx, &sysCtx)
-		if err == nil {
-			fmt.Println(out)
-		}
+		out, err := ShowLayerUsage(ctx, &sysCtx, url[0])
+		fmt.Print(out)
```

Comment:
Needs an error check before we print `out`.

Reply:
no, that's not necessary because we always return `""` when `err == nil`

Reply:
Yes, I saw that, but the Print is unnecessary when `err != nil`; we should do an early return.

Reply:
Not to mention this hampers readability, by delegating responsibility on the reader to figure out the return value of `out` from `ShowLayerUsage` when `err != nil`, it's clearly not obvious.